examples/standalone-with-org-policies/main.tf

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  key_rotation_period_seconds = "2592000s" #30 days
+  location                    = "us-east4"
+}
+
```

Comment:
are any of the workloads missing (i.e the Dataflow pipelines?)

Reply:
By design, this example should have no dataflow jobs or network configuration.